src/main/java/edu/uml/diet/DbParser.java

```diff
@@ -0,0 +1,607 @@
+                databaseFood.setZinc(Double.parseDouble(delimited[16]));
+                databaseFood.setCopper(Double.parseDouble(delimited[17]));
+                databaseFood.setManganese(Double.parseDouble(delimited[18]));
+                databaseFood.setSelenium(Double.parseDouble(delimited[19]));
+                databaseFood.setVit_C(Double.parseDouble(delimited[20]));
+                databaseFood.setThiamin(Double.parseDouble(delimited[21]));
+                databaseFood.setRiboflavin(Double.parseDouble(delimited[22]));
+                databaseFood.setNiacin(Double.parseDouble(delimited[23]));
+                databaseFood.setPanto_acid(Double.parseDouble(delimited[24]));
+                databaseFood.setVit_B6(Double.parseDouble(delimited[25]));
+                databaseFood.setFolate_Tot(Double.parseDouble(delimited[26]));
+                databaseFood.setFolic_acid(Double.parseDouble(delimited[27]));
+                databaseFood.setFood_Folate(Double.parseDouble(delimited[28]));
+                databaseFood.setFolate_DFE(Double.parseDouble(delimited[29]));
+                databaseFood.setCholine_Tot(Double.parseDouble(delimited[30]));
+                databaseFood.setVit_B12(Double.parseDouble(delimited[31]));
+                databaseFood.setVit_A_IU(Double.parseDouble(delimited[32]));
+                databaseFood.setVit_A_RAE(Double.parseDouble(delimited[33]));
+                databaseFood.setRetinol(Double.parseDouble(delimited[34]));
+                databaseFood.setAlpha_Carot(Double.parseDouble(delimited[35]));
+                databaseFood.setBeta_Carot(Double.parseDouble(delimited[36]));
+                databaseFood.setBeta_Crypt(Double.parseDouble(delimited[37]));
+                databaseFood.setLycopene(Double.parseDouble(delimited[38]));
+                databaseFood.setLutZea(Double.parseDouble(delimited[39]));
+                databaseFood.setVit_E(Double.parseDouble(delimited[40]));
+                databaseFood.setVit_D_mcg(Double.parseDouble(delimited[41]));
+                databaseFood.setVit_D_IU(Double.parseDouble(delimited[42]));
+                databaseFood.setVit_K(Double.parseDouble(delimited[43]));
+                databaseFood.setFA_Sat(Double.parseDouble(delimited[44]));
+                databaseFood.setFA_Mono(Double.parseDouble(delimited[45]));
+                databaseFood.setFA_Poly(Double.parseDouble(delimited[46]));
+                databaseFood.setCholestrl(Double.parseDouble(delimited[47]));
+                databaseFood.setGmWt_1(Double.parseDouble(delimited[48]));
+                databaseFood.setGmWt_Desc1(delimited[49]);
+                databaseFood.setGmWt_2(Double.parseDouble(delimited[50]));
+                databaseFood.setGmWt_Desc2(delimited[51]);
+                databaseFood.setRefuse_Pct(Double.parseDouble(delimited[52]));
+
+                databaseFoodList.add(databaseFood);
+            }
+
+            br.close();
+
+        }
+        catch (Exception e){
```

Comment:
another problematic exception handling issue here. 

also, replace this construct with try with resources